mail/dovecot/Makefile

```diff
@@ -65,6 +71,14 @@ else
 	--with-ldap=no
 endif
 
+ifneq ($(strip $(CONFIG_DOVECOT_SQLITE)),)
+  CONFIGURE_ARGS+= \
+        --with-sqlite=yes
+else
+  CONFIGURE_ARGS+= \
+        --with-sqlite=no
+endif
+
```

Comment:
Can be handled in CONFIGURE_ARGS directly, see https://git.lede-project.org/?p=source.git;a=blob;f=package/network/services/lldpd/Makefile;h=de7de03cc86af175f945bd9ad96e4b5637390e67;hb=HEAD#l101